crates/driver/src/infra/solver/dto/auction.rs

```diff
@@ -12,18 +12,23 @@ use {
         util::conv::{rational_to_big_decimal, u256::U256Ext},
     },
     app_data::AppDataHash,
+    ethcontract::H160,
     ethrpc::alloy::conversions::IntoLegacy,
     model::order::{BuyTokenDestination, SellTokenSource},
     std::collections::HashMap,
 };
 
+pub type WrapperCalls = HashMap<order::Uid, Vec<(H160, Vec<u8>, bool)>>;
```

Comment:
Then, this type can be simplified to
```suggestion
pub type WrapperCalls = HashMap<order::Uid, Vec<solvers_dto::auction::WrapperCall>>;
```
, or even better- dropped 😄